rt-google-embeds.php

```diff
@@ -19,7 +19,7 @@
 }
 
 if ( ! defined( 'RT_GOOGLE_EMBEDS_VERSION' ) ) {
-	define( 'RT_GOOGLE_EMBEDS_VERSION', '1.2' );
+	define( 'RT_GOOGLE_EMBEDS_VERSION', '1.1' );
```

Comment:
@SH4LIN Any reason to downgrade the version? 🤔

Reply:
I checked it last commit was mine and mistakenly pushed 1.2 here. So updated it.

Reply:
Okay, thank you for the details.